### PR TITLE
MINOR: Inline "Running a Kafka broker in KRaft mode" in README.md

### PR DESCRIPTION
**Pull request by ijuma**
Also moved KRaft mode above zk mode.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by mimaison — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -83,15 +83,17 @@ fail due to code changes. You can just run:
  
     ./gradlew processMessages processTestMessages
 
+### Running a Kafka broker in KRaft mode
+
+    KAFKA_CLUSTER_ID="$(./bin/kafka-storage.sh random-uuid)"
+    ./bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c config/kraft/server.properties
+    ./bin/kafka-server-start.sh config/kraft/server.properties
+
```
Comment (jsancio) [inline README.md:91]: Can we have a `###` heading with something like "Running a Kafka Broker" with 2 `####` heading, one for KRaft-mode and one for ZK-mode?

The `### Running a Kafka Broker` should mention that only one subsection should be followed. In other words the Kafka cluster should have one KRaft-mode server or ZK-mode server but not both.

**Review by jsancio — COMMENTED** [review]
Thanks for the fix @ijuma

File: README.md (hunk 1 above)
Comment (ijuma) [reply README.md:91]: For better or worse, the README is not structured as a quick start - you are not supposed to follow every step in it. Instead, it is meant as a useful set of things one may want to do. The thing you suggested doesn't fit with everything else we have in my opinion. What do you think?

**Comment by ijuma** [comment]
I went ahead and merged this since it's a clear improvement over what we have. We can do follow up PRs for further improvements.

**Pull request merged**